ui-speedspacechart/src/components/helpers/drawElements/powerRestrictions.ts

```diff
@@ -75,25 +75,25 @@ export const drawPowerRestrictions = (
     LINEAR_LAYERS_HEIGHTS.POWER_RESTRICTIONS_HEIGHT - LINEAR_LAYER_SEPARATOR_HEIGHT
   );
 
-  (powerRestrictions as SimulationPowerRestrictionRange[]).forEach((range, i) => {
-    if (range.handled) {
+  powerRestrictions.forEach(({ position, value }, i) => {
+    if (value.isCompatible) {
```

Comment:
It seems that isCompatible is equal to "handled". Can we use only "handled" and remove the isCompatible type ?

Reply:
good catch! done

Reply:
done